react-native-libraries.json

```
@@ -9607,5 +9607,13 @@
     "githubUrl": "https://github.com/huextrat/react-native-maps-routes",
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/monkin77/react-native-esper-sdk",
+    "nameOverride": "React Native Esper SDK",
+    "examples": [
+      "https://github.com/monkin77/react-native-esper-sdk/blob/master/example/src/App.tsx"
```

Hello @monkin77, thank you for adding new package to the directory! 👍 

Have one nit for the example app link, before we can ship it:

```suggestion
      "https://github.com/monkin77/react-native-esper-sdk/blob/master/example"
```

---

Thanks! just updated it